sdk/python/endpoints/online/managed/online-endpoints-simple-deployment.ipynb

```diff
@@ -136,50 +140,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Creating a local endpoint\n",
+    "# Define an endpoint name\n",
+    "endpoint_name = \"my-endpoint\"\n",
+    "\n",
+    "# Example way to define a random name\n",
     "import datetime\n",
     "\n",
-    "local_endpoint_name = \"local-\" + datetime.datetime.now().strftime(\"%m%d%H%M%f\")\n",
+    "endpoint_name = \"endpt-\" + datetime.datetime.now().strftime(\"%m%d%H%M%f\")\n",
     "\n",
     "# create an online endpoint\n",
     "endpoint = ManagedOnlineEndpoint(\n",
-    "    name=local_endpoint_name, description=\"this is a sample local endpoint\"\n",
+    "    name = endpoint_name,\n",
+    "    description = \"this is a sample online endpoint\",\n",
+    "    auth_mode = \"key\",\n",
+    "    tags = {\"foo\": \"bar\"},\n",
     ")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "ml_client.online_endpoints.begin_create_or_update(endpoint, local=True)"
-   ]
-  },
-  {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 2.2 Create local deployment\n",
+    "## 2.2 Define the deployment\n",
     "\n",
-    "The example contains all the files needed to deploy a model on an online endpoint. To deploy a model, you must have:\n",
+    "A deployment is a set of resources required for hosting the model that does the actual inferencing. To deploy a model, you must have:\n",
     "\n",
-    "* Model files (or the name and version of a model that's already registered in your workspace). In the example, we have a scikit-learn model that does regression.\n",
-    "* The code that's required to score the model. In this case, we have a score.py file.\n",
-    "* An environment in which your model runs. As you'll see, the environment might be a Docker image with Conda dependencies, or it might be a Dockerfile.\n",
-    "* Settings to specify the instance type and scaling capacity.\n",
+    "- Model files (or the name and version of a model that's already registered in your workspace). In the example, we have a scikit-learn model that does regression.\n",
+    "- A scoring script, that is, code that executes the model on a given input request. The scoring script receives data submitted to a deployed web service and passes it to the model. The script then executes the model and returns its response to the client. The scoring script is specific to your model and must understand the data that the model expects as input and returns as output. In this example, we have a *score.py* file.\n",
```

Comment:
Do we want to add a link to the scoring script documentation

Reply:
Thanks for the suggestion. We were thinking we already linked the how-to documentation at the top, and actually most explanations here are duplicate of it. So that seems a bit redundant.